RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ------
+* [*] Embed Block Bottom Sheet to set a link [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3436]
```

Comment:
Oh, my understanding is that the embed block is still behind the dev flag, right? If we keep it that the release notes mention will be confusing, right?

Reply:
Good point, yeah it's behind a dev flag so makes sense not to include it in the release notes 👍 .

I verified if we already included entries related to the embed block and I found this one:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/b5c31ba65e15680976a381016a1a722996883214/RELEASE-NOTES.txt#L39

So probably we should remove it too, wdyt?

Reply:
Hmm, yeah, let's remove them. Technically speaking, those were already published to the Stores, but better to remove from the release notes files. Thanks for reviewing the past entries!

Reply:
I've just removed all the entries in the release notes related to the embed block in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3436/commits/867a5dc1669fc4ff71ae06bf2b4b1f17a30e790c).